doc/source/whats_new.rst

```diff
@@ -31,6 +31,9 @@ Changelog
 
     - Add new object :class:`mne.decoding.TimeDecoding` for decoding sensors' evoked response across time by `Jean-Remi King`_
 
+    - Add support to append new channels from a list of Raw objects by `Chris Holdgraf`_
```

Comment:
So adding buffer_size_sec to Raw info fields only won't fix this error, since the evoked object being read in doesn't have a "buffer_size_sec" field and _merge_info checks for it. Is it kosher if I just manually add that field in before running the test? E.g.:

```python
evoked = read_evokeds(fname, condition=0)
evoked.info['buffer_size_sec'] = None
```
(I can confirm that this makes the tests for evoked pass)

Otherwise it seems like we'd need to modify _merge_info to not always check for all of those fields...